src/chia_log/log_consumer.py

```diff
@@ -217,40 +216,38 @@ def create_log_consumer_from_config(config: dict) -> Optional[LogConsumer]:
     enabled_consumer_config = config[enabled_consumer]
 
     if enabled_consumer == "file_log_consumer":
-        if not check_keys(required_keys=["file_path"], config=enabled_consumer_config):
+        if "file_path" not in enabled_consumer_config or not enabled_consumer_config["file_path"]:
```

Comment:
I believe it's actually impossible to fail in that way anymore, since not defining file_path will fall back to the default, i.e. `~/.chia/mainnet/log/debug.log`. The only way I found is if you provide an empty `chia_logs` block, which crashes and burns.